apps/sms/test/marionette/lib/views/shared/composer_accessors.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/* global module */
+
+var SELECTORS = Object.freeze({
+  messageInput: '#messages-input',
+  subjectInput: '.subject-composer-input',
+  sendButton: '#messages-send-button',
+  attachButton: '#messages-attach-button',
+  charCounter: '.message-counter',
+  mmsLabel: '.mms-label',
+  attachment: '#messages-input .attachment-container'
+});
+
+function ComposerAccessor(client) {
+  this.client = client;
+}
+
+ComposerAccessor.prototype = {
+  get messageInput() {
+    return this.client.helper.waitForElement(SELECTORS.messageInput);
+  },
+
+  get subjectInput() {
+    return this.client.findElement(SELECTORS.subjectInput);
+  },
+
+  get sendButton() {
+    return this.client.helper.waitForElement(SELECTORS.sendButton);
```

Comment:
it's never very clear to me the difference between waitForElement and findElement. Does findElement fail right away if it can't find the element ?